From c43c536651a220e9e4d8e82013fbea7032a8a6c2 Mon Sep 17 00:00:00 2001
From: Roman Bulgakov <roman.bulgakov@globallogic.com>
Date: Wed, 29 Jul 2015 21:57:34 +0300
Subject: [PATCH 161/172] FogBugz #315701-8: dts: add OCRAM EDAC entry

This patch adds the new OCRAM EDAC entry into the
Arria 10 DTSI file.

Signed-off-by: Roman Bulgakov <roman.bulgakov@globallogic.com>
Signed-off-by: Dinh Nguyen <dinguyen@opensource.altera.com>
[czou:Original patch taken from
https://github.com/altera-opensource/linux-socfpga.git socfpga-4.1]
Signed-off-by: czou <cao.zou@windriver.com>
---
 arch/arm/boot/dts/socfpga_arria10.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/socfpga_arria10.dtsi b/arch/arm/boot/dts/socfpga_arria10.dtsi
index fb28f90..f3493d0 100644
--- a/arch/arm/boot/dts/socfpga_arria10.dtsi
+++ b/arch/arm/boot/dts/socfpga_arria10.dtsi
@@ -595,6 +595,14 @@
 				     <0 0 IRQ_TYPE_LEVEL_HIGH>;
 		};
 
+		ocramedac@ff8c3000 {
+			compatible = "altr,a10-ocram-edac";
+			reg = <0xff8c3000 0x5000>;
+			iram = <&ocram>;
+			interrupts = <0 2 IRQ_TYPE_LEVEL_HIGH>,
+				     <0 0 IRQ_TYPE_LEVEL_HIGH>;
+		};
+
 		mmc: dwmmc0@ff808000 {
 			#address-cells = <1>;
 			#size-cells = <0>;
-- 
1.9.1

